integration-tests/database/index.ts

```diff
@@ -24,87 +24,95 @@ let rdsInstanceArgs: aws.rds.InstanceArgs | undefined;
 
 console.log(`Running test scenario #${testScenario}`);
 switch (testScenario) {
-    case 1:
-        // Happy Path for redshift cluster.
-        redshiftClusterArgs = {
-            clusterIdentifier: "test",
-            nodeType: "dc1.large",
-            allowVersionUpgrade: true,
-            logging: {
-                enable: true,
-                bucketName: "random bucket",
-            },
-            encrypted: true,
-            publiclyAccessible: false,
-        };
-        break;
-    case 2:
-        // Logging undefined, not encrypted, does not allow version upgrade
-        // and is publicly accessible for redshift cluster.
-        redshiftClusterArgs = {
-            clusterIdentifier: "test",
-            nodeType: "dc1.large",
-            allowVersionUpgrade: false,
-        };
-        break;
-    case 3:
-        // Happy Path for dynamodb.
-        dynamodbArgs = {
-            hashKey: "test",
-            attributes: [{
-                name: "test",
-                type: "S",
-            }],
-            serverSideEncryption: {
-                enabled: true,
-            },
-        };
-        break;
-    case 4:
-        // Dynamodb server side encryption disabled.
-        dynamodbArgs = {
-            hashKey: "test",
-            attributes: [{
-                name: "test",
-                type: "S",
-            }],
-            serverSideEncryption: {
-                enabled: false,
-            },
-        };
-        break;
-    case 5:
-        // RDS Instance happy path.
-        rdsInstanceArgs = {
-            instanceClass: "db.m5.large",
-            storageEncrypted: true,
-            multiAz: true,
-        };
-        break;
-    case 6:
-        // RDS Instance - no backup & no multiAz.
-        rdsInstanceArgs = {
-            instanceClass: "db.m5.large",
-            backupRetentionPeriod: 0,
-            publiclyAccessible: true,
-        };
-        break;
-    default:
-        throw new Error(`Unexpected test scenario ${testScenario}`);
+  case 1:
```

Comment:
Can you revert the indentation changes, i.e. keeping 4 spaces instead of 2 (in this file and the other ones in this PR that changed)?